.eslintrc.json

```diff
@@ -1,7 +1,7 @@
 {
     "root": true,
     "extends": [
-        "eslint-config-moxy/es5",
+        "eslint-config-moxy/es6",
```

Comment:
This is a breaking change because the min supported nodejs version will be >= 6 (technically 5 but that's no longer a lts). This means that we must edit the commit body to have `BREAKING CHANGE: Minimum required nodejs version is now v6.0.0`.

Reply:
For reference: https://conventionalcommits.org